www/components/Nav.js

```diff
@@ -61,14 +62,15 @@ const Nav = ({ drawerOpen, setDrawerOpen }) => {
             </Link>
           </div>
 
-          <Link href={`/user/${user._id}`} passHref>
-            <Button
-              text={user.username}
-              icon={<AccountCircleIcon />}
-              varient="icon"
-              size="small"
-            />
-          </Link>
+          <Button
+            text={user.username}
+            icon={<AccountCircleIcon />}
```

Comment:
Ooooh I like you the dropdown is passed into the button. Thats a really great API 👍.